src/pages/submissions/Submissions.jsx

```diff
@@ -67,7 +68,9 @@ class Submissions extends React.Component {
         }, doc.data());
         let collection = DBCollections.rejects;
         if (approval) {
-          collection = DBCollections.members;
+          // determine the right collection depending on applicant value
+          // can be either members or advertisers, set by SubcscriptionForm
+          collection = DBCollections[data.applicant];
```

Comment:
store either to members or advertisers collections depending on data.applicant field